Block/Js.php

```diff
@@ -774,4 +774,40 @@ public function isPreventSSOCustomersFromEditingAccountInformation()
     {
         return $this->featureSwitches->isPreventSSOCustomersFromEditingAccountInformation();
     }
+
+    /**
+     * Returns list of selectors that should match login and logout buttons/links
+     * to be converted to Bolt Custom SSO login buttons
+     *
+     * @return array of selectors
+     */
+    public function getCustomSSOSelectors()
+    {
+        $transformRouteToSelector = function ($route) {
+            return sprintf("[href^=\"%s\"]", rtrim($this->getUrl($route), '/'));
+        };
+
+        $additionalSelectors = $this->configHelper->getAdditionalCustomSSOSelectors();
+        return array_merge(
+            [
+                $transformRouteToSelector('customer/account/login') => [],
+                $transformRouteToSelector('customer/account/create') => [],
+                $transformRouteToSelector('customer/account/logout') => ['logout' => true],
```

Comment:
How does this fill effect live merchants, which already have something to make logout?

Reply:
@vitaliyreznikov this whole PR affect live merchants. They have all this logic implemented via code injected manually in the process of integration, through Magento blocks and widgets. The old integration needs to be removed.

Reply:
I meant if this affects live merchants if we do not remove the old integration?
They can update the plugin version by themself and we need to make sure that nothing is broken.

Reply:
I can set up both somewhere and check. But even if it had worked on that instance, it may break somewhere else. We can expect some issues, but they would be obvious and we'd have a solution at hand. We can also send messages to all merchants with active SSO, and let them know the update process - removing the block / widget.

Reply:
What do you think about making the change under feature switch then?

Reply:
@vitaliyreznikov disabled by default?

Reply:
One option we can enable by default by manually disable for all merchants who already on SSO.
So no additional steps for new SSO integations.

Reply:
@vitaliyreznikov, done adding a switch